src/api/searchApi.ts

```diff
@@ -216,6 +217,7 @@ export interface FetchResultsParams {
   [ResultParam.Results]?: number | null;
   [ResultParam.Series]?: string | null;
   [ResultParam.ScientificIndex]?: string | null;
+  [ResultParam.ScientificIndexStatus]?: string | null;
```

Comment:
burde denne også ha type? antar den støtter et fixed sett med argument
```suggestion
  [ResultParam.ScientificIndexStatus]?: ScientificIndexStatuses | null;
```